example/app.js

```diff
@@ -18,10 +20,12 @@ restServer.toggleLogging(); // logging is off by default, enable it
 fetchMock.mock('^http://localhost:3000', restServer.getHandler());
 
 const restClient = simpleRestClient('http://localhost:3000');
-const delayedRestClient = (type, resource, params) => new Promise((resolve, reject) => setTimeout(() => resolve(restClient(type, resource, params)), 1000));
+const delayedRestClient = (type, resource, params) => new Promise(resolve => setTimeout(() => resolve(restClient(type, resource, params)), 1000));
+
+const CustomLayout = withAppTitle('Example Admin')(Layout);
 
 render(
-    <Admin restClient={delayedRestClient}>
+    <Admin restClient={delayedRestClient} appLayout={CustomLayout}>
```

Comment:
Maybe we can add a `title` attribute to the `Admin` component to hide this complexity to the end user? Understanding HOCs just to change the title is a bit too much at that stage IMO.